lib/src/cli.rs

```diff
@@ -153,19 +157,32 @@ async fn container_import(repo: &str, imgref: &str) -> Result<()> {
     let import = crate::container::import(repo, &imgref, Some(tx_progress));
     tokio::pin!(import);
     tokio::pin!(rx_progress);
-    loop {
+    let import = loop {
         tokio::select! {
             _ = rx_progress.changed() => {
                 let n = rx_progress.borrow().processed_bytes;
                 pb.set_message(&format!("Processed: {}", indicatif::HumanBytes(n)));
             }
             import = &mut import => {
                 pb.finish();
-                println!("Imported: {}", import?.ostree_commit);
-                return Ok(())
+                break import?;
             }
         }
+    };
+
+    if let Some(write_ref) = write_ref {
+        repo.set_ref_immediate(
+            None,
+            write_ref,
+            Some(import.ostree_commit.as_str()),
+            gio::NONE_CANCELLABLE,
+        )?;
+        println!("Imported: {} => {}", write_ref, import.ostree_commit.as_str());
+    } else {
+        println!("Imported: {}", import.ostree_commit);
     }
+
+    return Ok(())
```

Comment:
Spurious `return` (I feel that the CI here does not run `clippy`).

Reply:
Huh I can't get clippy to warn about that.  You're right CI isn't, I'll do a followup PR to add that and fix the existing issues.

Reply:
Fixed